app/src/androidTest/java/com/github/onlynotesswent/ui/search/SearchTest.kt

```diff
@@ -0,0 +1,136 @@
+package com.github.onlynotesswent.ui.search
+
+import android.graphics.Bitmap
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import com.github.onlynotesswent.model.note.Note
+import com.github.onlynotesswent.model.note.NoteRepository
+import com.github.onlynotesswent.model.note.NoteViewModel
+import com.github.onlynotesswent.model.note.Type
+import com.github.onlynotesswent.ui.navigation.NavigationActions
+import com.github.onlynotesswent.ui.navigation.Screen
+import com.google.firebase.Timestamp
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.mockito.kotlin.any
+import org.mockito.kotlin.verify
+
+class SearchScreenTest {
+  private lateinit var navigationActions: NavigationActions
+  private lateinit var noteViewModel: NoteViewModel
+  private lateinit var noteRepository: NoteRepository
```

Comment:
Like Santiago said:
@mock annotation can be added
and the name should contain the word mock like mockNoteRepository, mockNavigationActions, etc..
Annotated mocks can then be initialized with MockitoAnnotations.openMocks(this).